core/webif.ml

```diff
@@ -124,19 +124,19 @@ struct
         Eval.apply render_cont valenv (t, []) >>= fun (_, v) ->
         let res = render_servercont_cont v in
         Lwt.return ("text/html", res)
-      | ClientReturn(cont, arg) ->
+      | ClientReturn (cont, arg) ->
         Debug.print("Doing ClientReturn for client ID " ^ client_id_str);
         Proc.resolve_external_processes arg;
         Eval.apply_cont cont valenv arg >>= fun (_, result) ->
         let json_state = generate_json_state req_data result in
         let result_json =
           Json.jsonize_value_with_state result json_state |> Json.json_to_string in
         Lwt.return ("text/plain", Utility.base64encode result_json)
-      | RemoteCall(func, env, args) ->
+      | RemoteCall (func, env, args) ->
         Debug.print("Doing RemoteCall for function " ^ Value.string_of_value func
           ^ ", client ID: " ^ client_id_str);
-        (* Debug.print ("func: " ^ Value.show func); *)
-        (* Debug.print ("args: " ^ mapstrcat ", " Value.show args); *)
+        Debug.print ("func: " ^ Value.show func);
```

Comment:
Do you want these debug messages to stay uncommented?

Reply:
I'll axe them! Thanks!